ironfist/src/cpp/combat/combat.cpp

```diff
@@ -390,6 +390,12 @@ void SpecialAttackGraphics(army *attacker, army *target)
 		v27 = 10;
 		v18 = 10;
 	}
+	if (attacker->creatureIdx == CREATURE_CYBER_BEHEMOTH) {
```

Comment:
When you start editing code like this, it's definitely time to start cleaning up. Can you maybe give good names to these four variables?